sonic-utilities-tests/feature_test.py

```diff
@@ -0,0 +1,165 @@
+import sys
+import os
+import mock
+from click.testing import CliRunner
+from swsssdk import ConfigDBConnector
```

Comment:
Suggest to reorganize imports per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports):

>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.

```
import os
import sys

import mock
from click.testing import CliRunner
from swsssdk import ConfigDBConnector
```